### PR TITLE
Add rujuta first file to branch

### PR DESCRIPTION
**Pull request by jml9904**
Hi Rujuta!

### CODE REVIEW

**Comment by rujutashinde** [comment]
- [ ] add

**Comment by yogeshpgarud** [comment]
👍 good one

File: rujutashinde-boston.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Boston, MA
```
Comment (yogeshpgarud) [inline rujutashinde-boston.md:1]: 👍

**Review by yogeshpgarud — APPROVED** [review]
approved 👍

**Pull request merged**